uptime-kuma/values.yaml

```diff
@@ -39,6 +39,18 @@ securityContext: {}
   # runAsNonRoot: true
   # runAsUser: 1000
 
+livenessProbe:
+  exec:
+    command:
+      - node
+      - extra/healthcheck.js
+
+readinessProbe:
+  httpGet:
+    path: /
+    port: http
+    scheme: HTTP
```

Comment:
not necessarily how I would have done it, but I suppose this provides the greatest flexibility